config/deploy/production.rb

```diff
@@ -8,6 +8,9 @@
 set :stage, :production
 set :branch, 'master'
 set :deploy_to, '/home/deploy/production'
+set :rails_env,      "production"
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.